app/router.js

```diff
@@ -6,7 +6,7 @@ const Router = Ember.Router.extend({
   rootURL: config.rootURL,
 });
 
-Router.map(function() {
+Router.map(function () {
```

Comment:
Because I rebased this branch against 016/master before pushing. I believe that diff was made in 016/master in b3dee2bbea7b7f111f4c7f745c3af5fd46e49953.